### PR TITLE
Document database addition (and do a step I missed last time)

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: docs/ops/postgres/add-database.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+push them into Vault.  That, however, would require that you restart
+everything with randomly-generated passwords, and that's a fairly
+disruptive operation, so you probably are better off manually injecting
+just your new password.
+
+* Consult ``1Password`` and retrieve the appropriate vault write token for
+  the instance you're working with from ``vault_keys.json``.
+* Set up your environment: ``export VAULT_ADDR=vault.lsst.codes ; export
+  VAULT_FORMAT=json ; export VAULT_TOKEN=<retrieved-token>``
+* Run ``vault kv get secret/k8s_operator/<instance>/postgres >
```
Comment (afausti) [inline docs/ops/postgres/add-database.rst:105]: I think you can simplify these steps by suggesting simply `vault kv patch secret/k8s_operator/<instance>/postgres exposurelog_password=<password>`

**Review by afausti — COMMENTED** [review]
It looks good. I like the detailed documentation. I just made a suggestion on the section about manually adding a secret to Vault.

File: docs/ops/postgres/add-database.rst (hunk 1 above)
Comment (athornton) [reply docs/ops/postgres/add-database.rst:105]: Oh!  Yes, that would be simpler than what I've been doing.

Comment (athornton) [reply docs/ops/postgres/add-database.rst]: Done.

**Pull request merged**